ec2/spark_ec2.py

```diff
@@ -643,12 +654,14 @@ def setup_cluster(conn, master_nodes, slave_nodes, opts, deploy_ssh_key):
 
     # NOTE: We should clone the repository before running deploy_files to
     # prevent ec2-variables.sh from being overwritten
+    repo_branch="{r} -b {b}".format(r=opts.spark_ec2_git_repo, b=opts.spark_ec2_branch)
+    print "Cloning spark-ec2 scripts from {rb} on master....".format(rb=repo_branch)
```

Comment:
I wanted to keep it closer to the actual clone command used. 
I don't know git well enough to be certain that it's always equivalent to the {r}/tree/{b} pattern.
But happy to change. Just let me know. 

fixed the ....